sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample6_CopyCustomModel.md

```diff
@@ -0,0 +1,68 @@
+# Copy a custom model between Form Recognizer resources
+
+This sample demonstrates how to copy a custom model between Form Recognizer resources.
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Copy a custom model
+There are several scenarios that require the models to be copied between Form Recognizer resources, like for example, to keep a backup of the created models.
+Copies can be made:
+- Within same Form Recognizer resource
+- Across other Form Recognizer resources that exist in any other supported region.
+
+For this sample, you will copy a model across Form recognizer resource. It assumes you have the credentials for both the source and the target Form Recognizer resources.
+
+## Creating the source and target `FormTrainingClient`
+
+To create a new `FormTrainingClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
```

Comment:
nit: I feel like adding the part about rotating the key is kind of out of scope for this sample

Reply:
good point. Will remove